src/stylus/product/product-details.styl

```diff
@@ -1,116 +1,90 @@
 
-.product_collection
-  margin 0 auto
-  max-width pageColumnWidth
+.product-info-container
+  padding-top 50px
+  margin-bottom 50px
   @media ipadsAndUp
-    margin 0 auto 50px auto
-
-.product_information-half
-  display inline-block
-  margin-bottom 30px
+    font-size 0
+    margin-left -32px
+.product-info__col
   vertical-align middle
   width 100%
   @media ipadsAndUp
-    width 49%
-
-.product_information-half-left
-  text-align center
-
-.product_information-image
-  max-width 300px
-  margin-bottom 30px
-
-.product_information-winner-details
-  margin-bottom 15px
-
-.winner-badge
-  background bestBeautyPink
-  color #fff
-  display inline-block
-  font-weight 700
-  letter-spacing 2.6px
-  margin-right 10px
-  padding 10px 15px
-  text-transform uppercase
-  vertical-align middle
-
-.product_information-winner-details .rating-box
-  display inline-block
-  vertical-align middle
+    display inline-block
+    font-size 16px
+    padding-left 32px
+    width 50%
 
-.product_information-brand
-.product_information-description
-.product_information-offer
-  font-weight 300
-
-.product_information-title
+.product-info__title
   margin-bottom 10px
-
-.product_information-brand
+.product-info__brand
   color #4e4e4e
   font-size 14px
-  margin-top 0
+  font-weight 300
   margin-bottom 25px
+  margin-top 0
 
-.product_information-finer-details
-  border-top 1px solid #DBDBDB
-  border-bottom 1px solid #DBDBDB
-  padding 15px 0
-  text-align right
+.product-info-meta
+  border-bottom 1px solid #dbdbdb
```

Comment:
Should we be using variables for colours? And what are your thoughts on putting the whole of "1px solid #color" behind a variable?